python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type_l3_port_channels.schema.yml

```diff
@@ -68,6 +68,30 @@ $defs:
                 type: str
                 description: |-
                   Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
+              rx_queue:
+                type: dict
+                description: |-
+                  Receive queue parameters for Receive Side Scaling (RSS) profile for this interface.
+                keys:
+                  count:
+                    type: int
+                    min: 1
+                    convert_types:
+                      - str
+                    description: |-
+                      Number of receive queues.
+                      The maximum value is platform dependent.
+                  worker:
+                    type: str
+                    description: |-
+                      Worker ids specified as combination of range and/or comma separated values
+                      such as 0-4,7.
+                  mode:
+                    type: str
+                    description: Mode applicable to the workers. Default mode is 'shared'.
```

Comment:
Set the default key instead of writing it in description. (for eos_designs we can have default :) )